### PR TITLE
Feature/ab2 d 5076 hpms bfd fail event

### PR DESCRIPTION
**Pull request by sb-keane**
**JIRA Tickets:**

[AB2D-5076](https://jira.cms.gov/browse/AB2D-5076) - Description

Deploy metrics lambda to prod
 
### What Does This PR Do?

Add sending a metrics event when hpms or bfd requests fail.

### Database Changes

- [X] All QuickSight impacts have been identified

### Security Implications

- [ ] This PR adds new software dependencies
- [ ] This PR modifies or invalidates our security controls
- [ ] This PR stores or transmits data that was not stored or transmitted before

- [ ] This PR requires additional review of its security implications for other reasons

### Submitter Checklist

- [X] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [X] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [X] Code checked for PHI/PII exposure

### CODE REVIEW

File: hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceImpl.java [hunk 1]
```diff
@@ -105,17 +107,26 @@ private void refreshToken(long currentTimestamp) {
...
     }
 
+    private void logEvents(String error) {
+        eventLogger.sendLogs(MetricsEvent.builder()
+                .service("HPMS")
+                .timeOfEvent(OffsetDateTime.now())
+                .eventDescription(String.format(error))
+                .stateType(MetricsEvent.State.CONTINUE)
+                .build());
+        eventLogger.log(EventClient.LogType.SQL,
```
Comment (sb-keane) [inline hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceImpl.java:126]: I can remove this one. I added this so we can track hpms errors but we don't really need to track it twice.

**Comment by sb-dkniesteadt** [comment]
Here is an example of what it looks like

File: hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceImpl.java (hunk 1 above)
Comment (sb-dkniesteadt) [reply hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceImpl.java:126]: Yes, sendLogs method already logs to sql.

**Review by sb-dkniesteadt — APPROVED** [review]
Approved if changes are made

**Review by sharshar-sb — DISMISSED** [review]

**Review by sharshar-sb — APPROVED** [review]

**Review by sb-dkniesteadt — APPROVED** [review]

**Pull request merged**